eisbuk-admin/package.json

```diff
@@ -43,11 +43,11 @@
     "start": "react-app-rewired start",
     "start:eisbuk": "REACT_APP_EISBUK_SITE=eisbuk.web.app NODE_ENV=production react-app-rewired start --reset-cache",
     "start:igorice": "REACT_APP_EISBUK_SITE=igorice.web.app NODE_ENV=production react-app-rewired start --reset-cache",
-    "emulators:inspect": "yarn --cwd functions run build && exec firebase emulators:start --project eisbuk --inspect-functions 9229",
-    "emulators:start": "yarn --cwd functions run build && exec firebase emulators:start --project eisbuk",
+    "emulators:inspect": "yarn --cwd functions functions:build && exec firebase emulators:start --project eisbuk --inspect-functions 9229",
```

Comment:
Is this prefix needed? We're in the `functions` directory already, so I fail to see why we need to specify it once again.

Reply:
No it is not...I thought about that as well...this is a bit more in detail, but yeah...